app/models/katello/rhsm_fact_name.rb

```diff
@@ -7,7 +7,11 @@ def set_name
     end
 
     def origin
-      'Redhat'
+      'RHSM'
+    end
```

Comment:
This is shown on the mouse "hover" tooltip, unsure if this can beak anything else tho.